invisible_cities/reco/corrections_new.py

```diff
@@ -200,10 +184,111 @@ def lt_xy_corrections(X : np.array, Y : np.array, Z : np.array, maps : ASectorMa
     LT  = get_maps_coefficient(X,Y)
     return correct_lifetime_(Z,LT)
 
-def apply_all_correction_single_maps(map_e0     : ASectorMap,
-                                     map_lt     : ASectorMap,
-                                     map_te     : Optional[ASectorMap] = None,
-                                     apply_temp : bool                 = True) -> Callable:
+def get_df_to_z_converter(map_te: ASectorMap) -> Callable:
+    """
+    For given map, it returns a function that provides the conversion
+    from drift time to z position using the mean drift velocity.
+
+    Parameters
+    ----------
+    map_te : AsectorMap
+        Correction map with time evolution of some kdst parameters.
+
+    Returns
+    -------
+    A function that returns z converted array for a given drift time input
+    array.
+    """
+
```

Comment:
Maybe we can raise exception if map_te.t_evol is None (which is the case of MC)

Reply:
I agree, but maybe mc should have the table in order to converge data and mc analysis.